src/amrex/extensions/MultiFab.py

```diff
@@ -1,13 +1,23 @@
 """
 This file is part of pyAMReX
 
-Copyright 2023 AMReX community
-Authors: Axel Huebl
+Copyright 2024 AMReX community
+Authors: Axel Huebl, David Grote
 License: BSD-3-Clause-LBNL
 """
 
+import numpy as np
+
 from .Iterator import next
 
+try:
+    from mpi4py import MPI as mpi
+
+    comm_world = mpi.COMM_WORLD
```

Comment:
While we have access to `amr`, we can replace `mpi.COMM_WORLD` with `amr.ParallelDescriptor.Communicator()` later on.

I did not expose that element yet, but we already created an mpi4py compatible `MPI_Communicator` for the MPMD logic in pyAMReX. This can be a follow-up PR.